### PR TITLE
Fix docs examples

### PR DESCRIPTION
**Pull request by sake92**
Improvements:
- openweathermap didnt work (needs an API key), so I used another free API :D
- websockets didnt work when using https, it has to be wss (secure socket)..
- 2 cosmetic changes

### CODE REVIEW

File: example/src/main/scala/example/Example.scala [hunk 1]
```diff
@@ -123,14 +123,10 @@ object XMLHttpRequest{
   @JSExport
   def main(pre: html.Pre) = {
     val xhr = new dom.XMLHttpRequest()
-    xhr.open("GET",
-      "http://api.openweathermap.org/" +
-      "data/2.5/weather?q=Singapore"
-    )
+    xhr.open("GET", "https://www.boredapi.com/api/activity")
```
Comment (sjrd) [inline example/src/main/scala/example/Example.scala:126]: These examples show up in the final document next to their output, and hence they only have ~40 characters a line. We should try to keep lines as short as possible:
```suggestion
    xhr.open("GET",
      "https://www.boredapi.com/api/activity")
```

File: example/src/main/scala/example/Example.scala [hunk 2]
```diff
@@ -123,14 +123,10 @@ object XMLHttpRequest{
...
     xhr.onload = { (e: dom.Event) =>
       if (xhr.status == 200) {
-        pre.textContent =
-          xhr.responseText
+        pre.textContent = xhr.responseText
```
Comment (sjrd) [inline example/src/main/scala/example/Example.scala:129]: Same problem here. The previous formatting was better for the short line width.

File: example/src/main/scala/example/Example.scala [hunk 3]
```diff
@@ -166,9 +162,7 @@ object AjaxExtension {
                 .ExecutionContext
                 .Implicits
                 .global
-    val url =
-      "http://api.openweathermap.org/" +
-      "data/2.5/weather?q=Singapore"
+    val url = "https://www.boredapi.com/api/activity"
```
Comment (sjrd) [inline example/src/main/scala/example/Example.scala:165]:
```suggestion
    val url =
      "https://www.boredapi.com/api/activity"
```

**Review by sjrd — CHANGES_REQUESTED** [review]
I have a few formatting comments, but otherwise LGTM.

**Comment by sake92** [comment]
Makes sense. All addressed now! @sjrd

**Review by sjrd — APPROVED** [review]

**Pull request merged**